test-cases/performance/bootstrap.yaml

```diff
@@ -0,0 +1,11 @@
+test_duration: 1440
+prepare_write_cmd: "cassandra-stress write cl=QUORUM n=550200300 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(200) n=FIXED(5)' -pop seq=1..550200300"
```

Comment:
@asias right now it's using 550GB.

Reply:
Do we start bootstrapping 4th node after the  prepare_write_cmd finishes? Do we need to set space_node_threshold?

It seems from the sct log in my run, the test from test-cases/features/repair-based-operations/longevity-100gb-4h-ReplaceNode.yaml did not wait for prepare_write_cmd tp fomosj before stress_cmd started.

Reply:
Here i3.3xlarge is used. In test-cases/features/repair-based-operations/longevity-100gb-4h-ReplaceNode.yaml and test-cases/features/repair-based-operations/longevity-1TB-12h-GrowShrink.yaml i3.4xlarge is used.

550GB is good enough in my view. We can start with this, get some results and later decide if we want to go for 1TB.